app/models/user.rb

```diff
@@ -0,0 +1,5 @@
+class User < ApplicationRecord
+  has_many :comments
+  has_many :posts
+  has_many :likes
+end
```

Comment:
- [ ] In the `user` class you should create a method that returns 3most recent post